setup/quickstart/halyard.md

```diff
@@ -9,4 +9,5 @@ We've written guides for a few environments/configurations to get you started
 with Spinnaker by using Halyard. If your desired environment matches one of 
 those listed below, we recommend following the linked guide. 
 
-* [Deploy Spinnaker to GKE](/guides/tutorials/codelabs/halyard-getting-started)
+* [Halyard on GKE Quickstart](/setup/quickstart/halyard-gke/)
+* [Halyard on GCE Quickstart](/setup/quickstart/halyard-gce/)
```

Comment:
![gke](https://cloud.githubusercontent.com/assets/4874941/26684287/83e91a62-46b4-11e7-9fdc-17d5854f0411.png)

I think GKE is used quite often outside of Google, but I'll switch both to their full name.